channel_dns.py

```diff
@@ -161,12 +162,12 @@ def look_for_source_data(self, token=None, value=None):
             linux_inotify        = re.compile('([A-Za-z0-9.-]*)\.L[0-9]{2}\.', re.IGNORECASE)
             dtrace_process       = re.compile('([0-9]+)\.([A-Za-z0-9-=]+)\.h\.([A-Za-z0-9.-=]+)\.c\.([A-Za-z0-9.-=]+)\.D1\.', re.IGNORECASE)
             dtrace_file_open     = re.compile('([0-9]+)\.([A-Za-z0-9-=]+)\.h\.([A-Za-z0-9.-=]+)\.f\.([A-Za-z0-9.-=]+)\.D2\.', re.IGNORECASE)
-            desktop_ini_browsing = re.compile('([^\.]+)\.([^\.]+)\.ini\.', re.IGNORECASE)
+            desktop_ini_browsing = re.compile('([^\.]+)\.([^\.]+)\.([^\.]+)\.ini\.', re.IGNORECASE)
```

Comment:
With regards to backward compat, could we change the regex from `([^\.]+)\` to `([^\.]*)\` because we want to accept desktop.ini tokens that dont send that extra param through

Reply:
Fixed.